adapter/src/main/java/com/twilio/kudu/sql/rules/KuduNestedJoinRule.java

```diff
@@ -30,28 +30,34 @@
 import org.apache.calcite.sql.SqlKind;
 import org.apache.calcite.tools.RelBuilderFactory;
 
-// @TODO: this rule is primed to be moved into our internal project and out of OSS.
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.List;
+
 public class KuduNestedJoinRule extends RelOptRule {
   public final static EnumSet<SqlKind> VALID_CALL_TYPES = EnumSet.of(SqlKind.EQUALS, SqlKind.GREATER_THAN,
       SqlKind.GREATER_THAN_OR_EQUAL, SqlKind.LESS_THAN, SqlKind.LESS_THAN_OR_EQUAL);
 
-  public static final int DEFAULT_BATCH_SIZE = 100;
+  public static final int DEFAULT_BATCH_SIZE = 5;
```

Comment:
Curious - Any reason to change the batch size ?

Reply:
was just for testing reverted